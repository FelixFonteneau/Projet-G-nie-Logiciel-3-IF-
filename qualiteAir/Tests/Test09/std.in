2
-8
34
1000
01/01/2017 00:00
01/01/1042 02:00
01/01/2017 02:00
7
