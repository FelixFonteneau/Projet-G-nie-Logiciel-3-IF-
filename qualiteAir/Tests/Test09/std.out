Bonjour, et bienvenue dans cette application de surveillance de la qualité de l'air.
Le service est en cours d'initialisation, veuillez patientez...

Nombre de mesures analysées : 20

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Quel est le rayon (en km) ? Format : 'nombres' et '.' seulement acceptés et radius > 0.


Quelle est la date (Au format JJ/MM/AAAA HH:MM) ?


Les quantités respectives de gaz/particules fines en moyenne dans la zone sont
(exprimées en microgrammes par mètre cube µg/m3):
Aucune mesure de O3 trouvée.
Aucune mesure de NO2 trouvée.
Aucune mesure de SO2 trouvée.
Aucune mesure de PM10 trouvée.


Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Merci et à bientôt !

